src/EditorFeatures/VisualBasicTest/Formatting/CodeCleanUpTests.vb

```diff
@@ -0,0 +1,368 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.CodeCleanup
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Diagnostics.VisualBasic
+Imports Microsoft.CodeAnalysis.Editing
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.Shared.Utilities
+Imports Microsoft.CodeAnalysis.SolutionCrawler
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Formatting
+
```

Comment:
```suggestion
```

Reply:
We allow but do not require a blank line at this position. Marking all such comments resolved to avoid noise in this pull request. Please feel free to modify the blank line analyzer in dotnet/roslyn-analyzers if you would like to see the repository standardize on a particular pattern.